Add gopkg.in/inf.v0 for benchmark comparison

SchumacherFM opened the pull request:
Bit faster than decimal in case >18 digits in PI. inf has high initial allocation costs.

```
goos: darwin
goarch: amd64
pkg: github.com/ericlagergren/decimal/benchmarks
BenchmarkPi_decimal_Go_9-4     	      30	  41574158 ns/op	 6720006 B/op	   70000 allocs/op
BenchmarkPi_decimal_Go_19-4    	       3	 465468668 ns/op	171520032 B/op	 3950000 allocs/op
BenchmarkPi_decimal_Go_38-4    	       1	1172683254 ns/op	421842192 B/op	 9080885 allocs/op
BenchmarkPi_decimal_Go_100-4   	       1	4205209252 ns/op	1510736184 B/op	26403268 allocs/op
BenchmarkPi_inf_9-4            	      20	  80067259 ns/op	39880076 B/op	  950000 allocs/op
BenchmarkPi_inf_19-4           	      10	 164385987 ns/op	76480150 B/op	 1850000 allocs/op
BenchmarkPi_inf_38-4           	       3	 337969916 ns/op	152600309 B/op	 3710000 allocs/op
BenchmarkPi_inf_100-4          	       1	1068472434 ns/op	458600928 B/op	10040001 allocs/op
```

ericlagergren commented:
sweet, would you mind updating the readme with the addition? otherwise, lgtm. thanks!!

SchumacherFM committed:
Add gopkg.in/inf.v0 for benchmark comparison

Bit faster than decimal in case >18 digits in PI:

goos: darwin
goarch: amd64
pkg: github.com/ericlagergren/decimal/benchmarks
BenchmarkPi_decimal_Go_9-4      	      30	  40170830 ns/op	 6720000 B/op	   70000 allocs/op
BenchmarkPi_decimal_Go_19-4     	       3	 435059247 ns/op	171521226 B/op	 3950002 allocs/op
BenchmarkPi_decimal_Go_38-4     	       1	1105309061 ns/op	421841136 B/op	 9080852 allocs/op
BenchmarkPi_decimal_Go_100-4    	       1	3883683881 ns/op	1510730552 B/op	26403187 allocs/op
BenchmarkPi_inf_9-4             	      20	  75110281 ns/op	39880083 B/op	  950000 allocs/op
BenchmarkPi_inf_19-4            	      10	 152512562 ns/op	76480169 B/op	 1850000 allocs/op
BenchmarkPi_inf_38-4            	       5	 323354597 ns/op	152600323 B/op	 3710000 allocs/op
BenchmarkPi_inf_100-4           	       1	1023068249 ns/op	458601040 B/op	10040000 allocs/op

SchumacherFM commented:
Readme updated! But the results for `inf` looks way too fast ...

ericlagergren commented:
It might really be that fast. Conforming to the decimal spec (properly handling Inf, NaN, etc.) adds a bit of overhead and the `inf` library seems pretty plain. But I'll double check.
